sihl-email/src/sihl_email.ml

```diff
@@ -392,16 +395,12 @@ module Queued
     else Job.dispatch_all emails
   ;;
 
-  let start () =
-    QueueService.register_jobs [ Sihl.Contract.Queue.hide Job.job ]
-    |> Lwt.map ignore
-  ;;
-
+  let start () = QueueService.register_jobs [ Sihl.Contract.Queue.hide Job.job ]
   let stop () = Lwt.return ()
 
   let lifecycle =
     Sihl.Container.create_lifecycle
-      Sihl.Contract.Email.name
+      Sihl.Contract.Email.name_queued
```

Comment:
@aronerben @aronerben The contract is not allowed to know implementation details such as queued emails.

Reply:
Done